lib/console.js

```diff
@@ -151,13 +151,12 @@ Console.prototype.timeEnd = function timeEnd(label) {
 
 
 Console.prototype.trace = function trace(...args) {
-  // TODO probably can to do this better with V8's debug object once that is
-  // exposed.
-  var err = new Error();
-  err.name = 'Trace';
-  err.message = util.format.apply(null, args);
-  Error.captureStackTrace(err, trace);
-  this.error(err.stack);
+  const o = {
```

Comment:
I think it's not really a typical error anymore but I do not have any strong feelings.